src/applications/disability-benefits/all-claims/content/form0781.jsx

```diff
@@ -4,12 +4,6 @@ import { formTitleTag, formTitle } from '../utils';
 export const form0781HeadingTag = 'VA FORM 21-0781';
 export const additionalFormsTitle = 'Additional Forms';
 
-export const form0781WorkflowChoices = {
```

Comment:
Created a separate content file for the workflow choice page as we've been doing with the other pages.